domains/transfer/Transfer.API/API.UnitTests/Claiming/Automation/ClaimServiceTests.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Claiming.Api.Models;
+using API.Claiming.Api.Repositories;
+using API.Claiming.Automation;
+using API.Claiming.Automation.Services;
+using FluentAssertions;
+using Google.Protobuf.WellKnownTypes;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using ProjectOrigin.Common.V1;
+using ProjectOrigin.WalletSystem.V1;
+using Xunit;
+using Attribute = ProjectOrigin.WalletSystem.V1.Attribute;
+
+namespace API.UnitTests.Claiming.Automation;
+
+public class ClaimServiceTests
+{
+    private static class Attributes
+    {
+        public const string AssetId = "AssetId";
+        public const string TechCode = "TechCode";
+        public const string FuelCode = "FuelCode";
```

Comment:
This is only used once in this test, so it can be inlined.

Reply:
corrected